### ISSUE TITLE
Change customAppBar from AppBar to PreferredSizeWidget 

### ISSUE THREAD

**Issue by rydmike** [rodydavis/storyboard#5]
The customAppBar property:
```
/// Override the default app bar of the storyboard
final AppBar customAppBar;
```

Should probably be a PreferredSizeWidget like in Scaffold:

```
/// An app bar to display at the top of the scaffold.
final PreferredSizeWidget customAppBar;
```

So that you can provide a totally custom app bar, instead of just an AppBar(). You can still give it an AppBar() when using PreferredSizeWidget, but also a totally custom version that implements PreferredSizeWidget itself.

**Comment by rodydavis** [comment]
Changed on version 0.1.0!

**Comment by rydmike** [comment]
Thanks, that did the trick. :)

I saw some other issue with the AppBar in the package after this fix too. 
Don't recall exactly what it was now. I noticed it on a clone I did for some experiments and I was using community https://pub.dev/packages/lint analyzer in that project, so it complained about a lot of things that I had to change before I could run build i anyway, but there was one, that was potentially an issue otherwise too. I'll see if I can find later again.

Getting a bit more familiar with the package now, so I'll probably make a fork for pull requests soon, but you are so fast that you will probably fix everything before I submit any requests anyway! ;)
